kfs-core/src/main/java/edu/arizona/kfs/sys/KFSPropertyConstants.java

```diff
@@ -11,6 +11,8 @@
     public static final String FINALIZED_DATE = "finalizedDate"; //This should really be a Rice property constant.
     public static final String BATCH_FILE_UPLOADS_BATCH_FILE_NAME = "batchFileName";
     public static final String BATCH_FILE_UPLOADS_FILE_PROCESS_TIMESTAMP = "fileProcessTimestamp";
+    public static final String INVENTORY_UNIT_CODE = "inventoryUnitCode";
+    public static final String INVENTORY_UNIT_ORGANIZATION_CODE = "inventoryUnitOrganizationCode";

```

Comment:
Since the actual BO class property name is organizationCode, we should change the value of this constant or use the standard organization code property constant in the BO class (line 86).